code/game/objects/items/storage/boxes/job_boxes.dm

```diff
@@ -45,10 +45,11 @@
 	if(HAS_TRAIT(SSstation, STATION_TRAIT_PREMIUM_INTERNALS))
 		new /obj/item/flashlight/flare(src)
 		new /obj/item/radio/off(src)
+		new /obj/item/condom_pack(src) //BUBBER EDIT: WEAR A CONDOM
```

Comment:
I believe this can be modular? Check `crafted` for early return and then check the trait